owner/manager.go

```diff
@@ -171,6 +180,10 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
 	logPrefix := fmt.Sprintf("[%s] %s ownerManager %s", m.prompt, m.key, m.id)
 	var err error
 	for {
+		if err != nil {
+			metrics.CampaignOwnerHistogram.WithLabelValues(logPrefix, err.Error()).Inc()
```

Comment:
Why not use counter? You just use method Inc().

Reply:
Sorry, I use the wrong name.